include/mbedtls/config.h

```diff
@@ -316,6 +316,13 @@
 //#define MBEDTLS_ENTROPY_HARDWARE_ALT
 
 /**
+ * \def MBEDTLS_ENTROPY_MIN_HARDWARE
```

Comment:
I think this is in the wrong place and should be under the subtitle `Entropy options`. It should also have a different  format with the descriptive text on the same line as the definition.